01/fuzzygwalchmei/wordvalue.py

```diff
@@ -0,0 +1,20 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    return [line.rstrip('\n') for line in open(DICTIONARY)]
```

Comment:
would strongly advice to use a context manager (`with`statement) who handles correctly closing files

Reply:
No need for `\n` in strip, strips default is to clear whitespace and linebreaks